.github/workflows/testing.yml

```diff
@@ -1,14 +1,14 @@
 on:
   push:
     branches:
-      - master
+      - main
```

Comment:
It would be sweet to also update this is the readme https://github.com/hashicorp/vagrant_cloud#releasing